services/sidecar/src/simcore_service_sidecar/config.py

```diff
@@ -37,6 +37,11 @@
 SIDECAR_DOCKER_VOLUME_LOG: str = os.environ.get(
     "SIDECAR_DOCKER_VOLUME_LOG", f"{SWARM_STACK_NAME}_log"
 )
+
+SIDECAR_HOST_HOSTNAME_PATH: Path = Path(
+    os.environ.get("SIDECAR_HOST_HOSTNAME_PATH", Path.home() / "hostname")
```

Comment:
There is no need to create yet another environ variable. I would simply assume
```python
SIDECAR_HOST_HOSTNAME_PATH = Path.home() / "hostname"

# soft way
if not SIDECAR_HOST_HOSTNAME_PATH.exists():
   log.error("Did you forget to mount hostname in ... ?  ..." )
# the hard way
if not SIDECAR_HOST_HOSTNAME_PATH.exists():
     raise ValueError(" ... ")

```

Reply:
BTW. do not use ``assert`` for production because they are all removed with PYTHON_OPTIMIZE ...

Reply:
this comes from outside, so I prefer having a variable